azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -437,13 +440,24 @@ private Set<Integer> getStaleContainers(final Duration containerValidity) throws
       // Select pods only from current Azkaban cluster and namespace
       final String label =
           CLUSTER_LABEL_NAME + "=" + this.clusterName + "," + APP_LABEL_NAME + "=" + POD_APPLICATION_TAG;
-      final V1PodList podList= this.coreV1Api.listNamespacedPod(this.namespace, null,
-          null, null, null, label,
-          null, null, null, null);
+      final V1PodList podList= this.coreV1Api.listNamespacedPod(
```

Comment:
What's the new argument added to this call as part of the api version change ?

Reply:
resourseVersionMatch